packages/ui/src/components/Channel/ChatScroll.tsx

```diff
@@ -128,6 +162,20 @@ export default function ChatScroll({
         inverted
         onScrollToIndexFailed={handleScrollToIndexFailed}
       />
+      <Modal
+        visible={activeMessage !== null}
+        onDismiss={() => setActiveMessage(null)}
+      >
+        {activeMessage !== null && (
+          <ChatMessageActions
+            currentUserId={currentUserId}
+            post={activeMessage!}
```

Comment:
Do we need the `!` here? I would think the null check would be enough, and if it's not there might be a case where it's an incorrect value that we should be handling.